packages/wdio-applitools-service/package.json

```diff
@@ -35,7 +35,8 @@
   },
   "dependencies": {
     "@wdio/eyes.webdriverio": "^2.0.2",
-    "@wdio/logger": "^5.4.6"
+    "@wdio/logger": "^5.4.6",
+    "geckodriver": "1.14.1"
```

Comment:
bootstrap and CI are failing because of new geckodriver release, had to hardcode it here. See [vladikoff/node-geckodriver#59](https://github.com/vladikoff/node-geckodriver/issues/59)

Reply:
This change needs to be reverted and fixed in a different PR